src/documents/InclusiveMakingAndHacking.html.md.handlebars

```diff
@@ -0,0 +1,205 @@
+
+* Plan, execute and review in the open; give people opportunity to provide feedback before and after the event or activity. A living document supported by technology such as a wiki can help to invite necessary commentary and revision.
+
+#### On STEM Diversity Issues
+
+Because hacking and making are strongly associated with STEM (Science, Technology, Engineering and Math) disciplines, it's not possible to discuss this subject without reference to the widespread work that has gone on to investigate and address the issue of demographic diversity in STEM both academically and <a href="https://www.wired.com/brandlab/2015/05/5-numbers-explain-stem-diversity-matters-us/" target="_blank" class="link-external">in the wider world</a>.
+
+The STEM acronym itself has come under some criticism for excluding art and design, leading to initiatives such as the <a href="http://stemtosteam.org/" target="_blank" class="link-external">STEM to STEAM Initiative</a>.
+
+We encourage those considering hacking and making in learning contexts to be aware of larger issues of diversity in STEM subjects, workplaces and communities, as these issues can have relevance and knock-on effects to the diversity and inclusion of hackathons and hacking.
+
+## Tools
+
+The sections below include examples of tools (software, hardware and design-related) that can be useful in inclusive design hacking. Some are early-stage research projects, while others are mature and in widespread usage in and outside the classroom.
+
+We have tried to highlight tools that fall somewhere at the nexus of openness, active development, strong communities, and adaptability. Separately or in combination, they might be used to:
+
+* prototype new ways of interacting with the digital or physical world that suit an individual's particular needs.
+* adapt and transform existing technology to support new uses.
+
+### Research From Prosperity 4All
+
+Floe participates in the <a href="http://www.prosperity4all.eu/" target="_blank" class="link-external">Prosperity 4All</a> project that has supported the development and cataloguing of tools relevant to inclusive learning through hacking and making. These include:
+
+#### The Nexus
+
+The <a href="https://github.com/GPII/nexus" target="_blank" class="link-external">Nexus</a> is an integration technology based on the <a href="https://github.com/fluid-project/infusion" target="_blank" class="link-external">Infusion</a> Javascript framework that allows <a href="https://wiki.gpii.net/w/Nexus_API" target="_blank" class="link-external">distributed systems to be connected together through shared and transformed state over HTTP and WebSockets protocols, as described in detail here.</a>
```

Comment:
I feel like there's something missing about this description of the Nexus—you mention the technology, but less so what it's _for_. I wonder if, with @simonbates' help (or by borrowing from the link above), we could try to situation how the Nexus is important for making something new possible. Yesterday at the community meeting, Simon said something about how "it lets you connect together systems and components that weren't designed to work together, and that might otherwise be incompatible" or something to that effect, and to me this is a powerful description of why the Nexus is needed for these kinds of projects. Equally, it should be said that the Nexus helps to support the creation of new assistive technologies that can reach deeper into the semantics and state of an application.